### PR TITLE
added multiple and offset to blue marble height to make it similar lo…

### PR DESCRIPTION
**Pull request by micahnyc**
…oking to esri terrain

### CODE REVIEW

**Comment by alexanderbock** [comment]
Do we know _why_ the layers are misaligned?  The Earth reference ellipsoid is very well defined and it seems like we could fix this a better way than hacking a few correction values

**Comment by micahnyc** [comment]
> 

Not sure, I'm guessing the layer comes from this page: https://visibleearth.nasa.gov/images/73934/topography 

It states "Data in these images were scaled 0-6400 meters. " So maybe a multiplier of 64 instead of 40?

**Comment by alexanderbock** [comment]
Ouh.. that's weird.  Maybe we should reach out to them what they mean exactly to do the correct thing?  Since there are heights on earth <0 and >6400

**Comment by alexanderbock** [comment]
:+1:

File: data/assets/scene/solarsystem/planets/earth/layers/heightlayers/blue_marble_height.asset [hunk 1]
```diff
@@ -12,7 +12,11 @@ local layer = {
   Identifier = "Earth_Bluemarble_Height",
   Enabled = asset.enabled,
   FilePath = texturesPath .. "earth_bluemarble_height.jpg",
-  Description = "Topographic layer from Blue Marble Next Generation"
+  Description = "Topographic layer from Blue Marble Next Generation",
+  Settings = {
+    Multiplier = 40,
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/earth/layers/heightlayers/blue_marble_height.asset:17]: Adding a comment where the numbers come from

**Pull request merged**